### PR TITLE
You can submit a folder path with the /Fo option. 

### PR DESCRIPTION
**Pull request by mhandb**
In this case CClash looked for the OBJ_FOLDER\.obj file. Now it will look for the OBJ_FOLDER\SOURCE_FILE_NAME.obj.
Fix issue #7

### CODE REVIEW

**Comment by inorton** [comment]
There are quite a few whitespace changes here that detract from what's actually being changed.

File: CClash/Compiler.cs [hunk 1]
```diff
@@ -477,8 +478,10 @@ public bool ProcessArguments(string[] args)
 
                         case "/Fo":
                             ObjectTarget = Path.Combine(WorkingDirectory, full.Substring(3));
-                            if (!Path.GetFileName(ObjectTarget).Contains("."))
+                            ObjectTarget = ObjectTarget.Replace('/', '\\');
```
Comment (inorton) [inline CClash/Compiler.cs]: I can't see the reason for this.  If this is about coping with cygwin paths that shouldn't really happen here.

Comment (inorton) [inline CClash/Compiler.cs]: I'd probably accept this if there were a unit test for it

**Comment by statico** [comment]
This is a friendly bump -- I'd love to see this merged since it fixes cclash for us.

**Comment by inorton** [comment]
Hiya, I'd be happy to accept this if there was a unit test

File: CClash/Compiler.cs (hunk 1 above)
Comment (inorton) [inline CClash/Compiler.cs]: Still looks suspiciously like cygwin support without any comments

**Comment by inorton** [comment]
Hiya, I've made some changes on the posixpaths branch that fix this bug

**Pull request closed without merging**